src/node_wasm_web_api.h

```diff
@@ -0,0 +1,54 @@
+#ifndef SRC_NODE_WASM_WEB_API_H__
+#define SRC_NODE_WASM_WEB_API_H__
```

Comment:
```suggestion
#ifndef SRC_NODE_WASM_WEB_API_H_
#define SRC_NODE_WASM_WEB_API_H_
```
We usually use just a single `_` at the end in other header files.